src/Security/MemberAuthenticator/ChangePasswordHandler.php

```diff
@@ -195,6 +196,8 @@ public function changePasswordForm()
      * @param array $data The user submitted data
      * @param ChangePasswordForm $form
      * @return HTTPResponse
+     * @throws \SilverStripe\ORM\ValidationException
+     * @throws \Psr\Container\NotFoundExceptionInterface
```

Comment:
Just some docblocks that needed to be added to shut up PHPStorm :)

Reply:
Please use `use` to simplify these FQN please.

Reply:
👍